main.py

```diff
@@ -71,7 +71,7 @@ def train_step(inputs,labels):
                 train_loss.reset_states()
                 train_accuracy.reset_states()
 
-        
+    save_model(model, epoch)
```

Comment:
Seems to be an indent missing

Reply:
No because we want to save the model once it is trained, after the last epoch.

Reply:
ok, miss-understood the commit message